cicecore/cicedyn/analysis/ice_history_mechred.F90

```diff
@@ -380,6 +407,12 @@ subroutine accum_hist_mechred (iblk)
          if (f_opening(1:1) /= 'x') &
              call accum_hist_field(n_opening, iblk, opening(:,:,iblk), a2D)
 
+         if (f_sirdgconc(1:1)/= 'x') &
+             call accum_hist_field(n_sirdgconc,   iblk, &
+                             aice(:,:,iblk)*(c1 - trcr(:,:,nt_alvl,iblk)), a2D)
+         if (f_sirdgthick(1:1)/= 'x') &
+             call accum_hist_field(n_sirdgthick,   iblk, &
+                             vice(:,:,iblk)*(c1 - trcr(:,:,nt_vlvl,iblk)), a2D)
```

Comment:
This doesn't look like the "total volume of ridged sea ice divided by area of ridges" in line 251 - it's not divided by an area